RawWaterManagement/Spillway/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,146 @@
+      "coordinates": [
+        -26.363813,
+        -95.138688
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Treat rather scientist. Financial chair hour explain market admit.",
+      "addressLocality": "Each laugh agent tough commercial candidate great. Against second whether figure fear able. However think body paper natural bed.",
+      "addressRegion": "Local significant thing. Information teach research offer wrong mother arrive. Address effect sort run some more president.",
+      "addressCountry": "Low government tough arrive task democratic. Nature kind popular next agent.",
+      "postalCode": "Take three local bit country focus. Entire plant nice look discussion.",
+      "postOfficeBoxNumber": "Huge benefit fact many ask program. Firm condition woman writer follow billion.",
+      "areaServed": "Better now doctor agree. Together measure increase read. Want life reach red."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Ever must sort where culture room at. Person popular side simply yourself much gun dinner."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:HDWO:33110928",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1993-03-09T19:34:50Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2006-06-14T21:09:34Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "State late teach word. Throw political music."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Very red could catch bit better. Defense sell mother. Agency lead character arm drop country."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Who usually similar yeah from report few."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Less foot defense mention price work anyone. Draw young my result. Senior life long system science respond such."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Person season available. Send church food. Reduce accept enjoy lead."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:EUUN:79828978",
+      "urn:ngsi-ld:dataModel:items:REVJ:43009644"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:ATVV:58241235",
+      "urn:ngsi-ld:dataModel:items:IVLM:48258577"
+    ]
+  },
+  "type": "Spillway",
+  "tag": {
+    "type": "Property",
+    "value": "Mean white admit father. Over especially sign event he sister. Hard down difference system organization accept. The half cause sure nothing fill."
+  },
+  "spillwayType": {
+    "type": "Property",
+    "value": "Broad-Crested"
+  },
+  "crestElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "crestLength": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "spillwayWidth": {
```

Comment:
Use of unitcode is encouraged to ensure consistency of collected data. Otherwise, no one knows if e.g. a temperature is expressed in °C or K 
The specification recommends to take it from UN CEFACT (https://unece.org/DAM/cefact/recommendations/rec20/rec20.zip)